### PR TITLE
Reworked text/font css on DiscoverRoversPage.tsx and placed cards in …

### PR DESCRIPTION
**Pull request by Espe-on**
…flexbox

now scales well and text stays inside it's parent card

### CODE REVIEW

**Comment by Espe-on** [comment]
please ignore me saying out loud that this doesn't work, I had the staging build open in Firefox

File: src/Pages/DiscoverRoversPage/DiscoverRoversPage.scss [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
   display: flex;
   flex-direction: column;
   align-items: center;
   text-align: center;
+  font-size : large;
+
   h1 {
-    font-size: 2.5rem;
+    font-size: larger;
   }
```
Comment (MichaelWalker) [inline src/Pages/DiscoverRoversPage/DiscoverRoversPage.scss:10]: Why do we not want to just use the global defaults here?

**Review by MichaelWalker — APPROVED** [review]
I'm a little surprised that we can't just use the default styling for an `h1`.
(would make things a little more consistent)

but if there is a good reason for it then I'm happy

File: src/Pages/DiscoverRoversPage/DiscoverRoversPage.scss (hunk 1 above)
Comment (Espe-on) [reply src/Pages/DiscoverRoversPage/DiscoverRoversPage.scss:10]: the styling for h1 is far too small

**Pull request merged**